app/src/main/java/com/infomaniak/mail/ui/main/folder/TwoPaneFragment.kt

```diff
@@ -48,9 +53,12 @@ abstract class TwoPaneFragment : Fragment() {
             else -> throw IllegalStateException()
         }
 
+    open fun doAfterFolderChanged() {}
+
+    fun isOnlyOneShown() = slidingPaneLayout.isSlideable
     fun areBothShown() = !slidingPaneLayout.isSlideable
```

Comment:
not a fan of this 2 functions being a duplicate

Reply:
Do you prefer this ?
```kt
fun isOnlyOneShown() = slidingPaneLayout.isSlideable
fun areBothShown() = !isOnlyOneShown()
```
🤷‍♂️